src/@next/Bar/BarStyle.ts

```diff
@@ -0,0 +1,93 @@
+import styled from 'styled-components';
+import { Device, ScreenSize } from '../..';
+import { Neutral } from '../utilities/colors';
+import { space8 } from '../utilities/spacing';
+
+export const StyledBar = styled.div`
+  display: flex;
+  flex-direction: row;
+  padding: 20px 0;
```

Comment:
I think this is also dynamic based on screen size. can @chiahou confirm the spacing?